api/src/main/kotlin/com/openlattice/collaborations/CollaborationsApi.kt

```diff
@@ -1,97 +1,189 @@
+        const val DATA_SET_ID_PATH = "/{$DATA_SET_ID_PARAM}"
+        const val ORGANIZATION_ID_PARAM = "organizationId"
+        const val ORGANIZATION_ID_PATH = "/{$ORGANIZATION_ID_PARAM}"
     }
 
+    /**
+     * Gets all [Collaboration] objects the caller has [Permission.READ] on.
+     *
+     * @return a list of [Collaboration] objects
+     */
     @GET(BASE)
     fun getCollaborations(): Iterable<Collaboration>
 
-    @POST(BASE)
-    fun createCollaboration(@Body collaboration: Collaboration): UUID
+    /**
+     * Gets the [Collaboration] object with the given collaboration id. The caller must have [Permission.READ] on the
+     * target [Collaboration] object.
+     *
+     * @param collaborationId [Collaboration] id
+     * @return the target [Collaboration] object
+     */
+    @GET(BASE + COLLABORATION_ID_PATH)
+    fun getCollaboration(@Path(COLLABORATION_ID_PARAM) collaborationId: UUID): Collaboration
 
-    @GET(BASE + ID_PATH_PARAM)
-    fun getCollaboration(@Path(ID) id: UUID): Collaboration
+    /**
+     * Gets all [Collaboration] objects the caller has [Permission.READ] on that include the given organization id. The
+     * caller must have [Permission.READ] on the target [Organization] object.
+     *
+     * @param organizationId [Organization] id
+     * @return a list of [Collaboration] objects
+     */
+    @GET(BASE + ORGANIZATIONS_PATH + ORGANIZATION_ID_PATH)
+    fun getCollaborationsWithOrganization(@Path(ORGANIZATION_ID_PARAM) organizationId: UUID): Iterable<Collaboration>
 
-    @GET(BASE + ORGANIZATIONS_PATH + ORGANIZATION_ID_PATH_PARAM)
-    fun getCollaborationsIncludingOrganization(@Path(ORGANIZATION_ID) organizationId: UUID): Iterable<Collaboration>
+    /**
+     * Creates a new [Collaboration] object.
+     *
+     * @param collaboration the [Collaboration] object to create
+     * @return the id of the newly created [Collaboration] object
+     */
+    @POST(BASE)
+    fun createCollaboration(@Body collaboration: Collaboration): UUID
 
-    @DELETE(BASE + ID_PATH_PARAM)
-    fun deleteCollaboration(@Path(ID) id: UUID)
+    /**
+     * Deletes the [Collaboration] object with the given collaboration id. The caller must have [Permission.OWNER] on
+     * the target [Collaboration] object.
+     *
+     * @param collaborationId [Collaboration] id
+     */
+    @DELETE(BASE + COLLABORATION_ID_PATH)
+    fun deleteCollaboration(@Path(COLLABORATION_ID_PARAM) collaborationId: UUID)
 
-    @POST(BASE + ID_PATH_PARAM + ORGANIZATIONS_PATH)
-    fun addOrganizationIdsToCollaboration(@Path(ID) id: UUID, @Body organizationIds: Set<UUID>)
+    /**
+     * Adds the given organization ids to the [Collaboration] object with the given collaboration id. The caller must
+     * have [Permission.OWNER] on the target [Collaboration] object.
+     *
+     * @param collaborationId [Collaboration] id
+     * @param organizationIds a set of [Organization] ids
+     */
+    @PATCH(BASE + COLLABORATION_ID_PATH + ORGANIZATIONS_PATH)
```

Comment:
just checking that the @POST to @PATCH change was intentional

Reply:
Yes, it's an intentional change. The reason I'm switching it is because `PATCH` seems more appropriate (I think) since it's meant to be used for applying partial updates to an existing thing vs `POST` is more for creating new things.